src/content/app/tools/blast/components/blast-views-navigation/BlastViewsNavigation.scss

```diff
@@ -0,0 +1,34 @@
+.blastViewsNavigation {
+  display: grid;
+  grid-template-columns: [left] auto [right] 1fr;
+  height: 74px; // this isn't ideal; but 100% won't work because the parent container only has a min-height
```

Comment:
Am confused with the `100 % won't work` part? Does it mean this will never work or won't work sometimes?

Reply:
Compare [this jsfiddle](https://jsfiddle.net/9fdnwvmh) with [this one](https://jsfiddle.net/9fdnwvmh/1/). In the first fiddle, the parent container (blue) has a fixed `height` property; and the child component (pink), having a height of 100%, is as high as the parent. In the second fiddle, the parent only has a `min-height` property; and the child's "100%" has nothing to reference; so the child component remains small.

It's the same as we are seeing here.

One way around this would be to change the parent component's (in our case, `TopBar`) display property to flex, so that its child stretches vertically. Perhaps this is what I should do; but then, maybe it's for a different PR.